golang/outbound.go

```diff
@@ -56,12 +56,17 @@ func (c *Connection) beginCall(ctx context.Context, serviceName string) (*Outbou
 		return nil, err
 	}
 
+	headers := callHeaders{
+		CallerName: c.localPeerInfo.ProcessName,
```

Comment:
We manually pass the service name to the hyperbahn client. I.e. The application developer does it

All node services implement two service names, ring pop and my-app.

We just make the application developers do more configuration.